Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/ConceptSearchParameters.java

```diff
@@ -0,0 +1,129 @@
+package edu.asu.conceptpower.servlet.rest;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+public class ConceptSearchParameters {
+
+    private String type_uri;
+    private String operator;
+    private Integer page;
+    private String equals_to;
+    private String similar_to;
+    private String description;
+    @NotEmpty(message = "Word cannot be empty.")
+    private String word;
+    private String pos;
+    private String concept_list;
+    private String type_id;
+    private String creator;
+    private String modified_by;
+    private Integer number_of_records_per_page;
```

Comment:
you probably want to validate a little bit more than just the word. what for example about pos value or operator?